modules/sample-modules/example-hooks/actionValidateOrder.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "actionValidateOrder"
```

Comment:
FYI you don't usually need quotes for strings in yaml unless you're using special characters, integers or `true|false`